apprise/plugins/NotifyGoogleChat.py

```diff
@@ -257,38 +316,51 @@ def parse_url(url):
         us to re-instantiate this object.
 
         Syntax:
-          gchat://workspace/webhook_key/webhook_token
+          gchatb://workspace/webhook_key/webhook_token
```

Comment:
`gchatb://` ?

Reply:
Typo from when I was testing implementation on a secondary plugin GoogleChatBis (gchatb). It's been removed.